### DISCUSSION TITLE
Expired ssl certificate

### DISCUSSION THREAD

**Discussion by 3a1b2c3** [vercel/next.js#49142] [Help] [answered]
A week ago my domains ssl certificate expired, i checked stuff explained here https://vercel.com/guides/domain-not-generating-ssl-certificate 
No reply from support. What can i do? Upload an external certificate?
Link sharing is brocken now

Result for www.journeai.com/CAA with DNSSEC validation and without DNSSEC detail:

{
"Status": 0 /* NOERROR */,
"TC": false,
"RD": true,
"RA": true,
"AD": false,
"CD": false,
"Question": [
{
"name": "www.journeai.com.",
"type": 257 /* CAA */
}
],
"Answer": [
{
"name": "www.journeai.com.",
"type": 5 /* CNAME */,
"TTL": 3600,
"data": "cname.vercel-dns.com."
},
{
"name": "cname.vercel-dns.com.",
"type": 257 /* CAA */,
"TTL": 600,
"data": "0 issue \"letsencrypt.org\""
},
{
"name": "cname.vercel-dns.com.",
"type": 257 /* CAA */,
"TTL": 600,
"data": "0 issue \"globalsign.com\""
},
{
"name": "cname.vercel-dns.com.",
"type": 257 /* CAA */,
"TTL": 600,
"data": "0 issue \"sectigo.com\""
}
],
"Comment": "Response from 2a00:edc0:6259:7:13::2."
}

**Comment by 3a1b2c3** [comment]
![image](https://user-images.githubusercontent.com/74843139/236058457-4ef0232d-80ed-4405-8b3f-af7956294c83.png)

**Comment by 3a1b2c3** [comment]
3 weeks in. What could i try?

**Reply by smaeda-ks** [reply]
@3a1b2c3 Please add the domain `www.journeai.com` explicitly to your project domains. You currently only have its Apex domain. Adding the domain will automatically trigger cert issuance for the `www.` subdomain

**Reply by 3a1b2c3** [reply]
thanks, do just add something like www.journeai.com/* to google registrar? Does vercel have any examples?

**Reply by smaeda-ks** [reply]
@3a1b2c3 you just need to add it to your Vercel project

**Comment by 3a1b2c3** [comment]
Where and how would i do it? this is the project settings. Is this a good source https://vercel.com/docs/concepts/projects/domains/add-a-domain?
i want to use links like this again https://www.journeai.com/?link=o_b64dc03ee3ae64f26f28a1ece254c14e&lang=English
![image](https://github.com/vercel/next.js/assets/74843139/1465c6b0-6a97-4ce2-a852-ad5c2d31be18)

**Reply by 3a1b2c3** [reply]
takes the site offline when i enter *.journeai.com in domain setting.  

DNS record was changed  yesterday as requested. Still says pending verification/. How long will it be offline if i do?
"vc-domain-verify=journeai.com,99bae6c6c0bd78bd4c55"
"9f923411a76a1e6ccdd0"

**Reply by smaeda-ks** [reply]
@3a1b2c3 You need to add `journeai.com` AND `www.journeai.com` separately. You can add multiple domains to a single project.

Using wildcard requires Vercel name servers:
https://vercel.com/docs/concepts/projects/domains/troubleshooting#configuring-nameservers-for-wildcard-domains

**Reply by smaeda-ks** [reply, marked as answer]
ok, part of the issue, you added the domain to your Hobby account, but you're trying to use that domain from your Pro team account. That's causing this verification. You need to "move" the domain from one to another, see: https://vercel.com/docs/concepts/projects/domains/transfer-your-domain

**Comment by 3a1b2c3** [comment]
Thanks again, I added the domain before upgrading the account. Honestly the whole account and team stuff adds complexity.
How on earth is any of this related to ssl certificate that worked fine until 3 weeks ago?

**Comment by 3a1b2c3** [comment]
4 weeks in new certificate, amen

**Reply by 3a1b2c3** [reply]
New settings kill my google email forwarding you have to pick you poisson?
